### ISSUE TITLE
BUG: documented CompareMaskedEqual value order incorrect

### ISSUE THREAD

**Issue by debfx** [seccomp/libseccomp-golang#49]
The libseccomp-golang documentation says:

```
    // CompareMaskedEqual returns true if the argument is equal to the given
    // value, when masked (bitwise &) against the second given value
    CompareMaskedEqual ScmpCompareOp = iota
```

However the libseccomp man page has:

```
       SCMP_CMP_MASKED_EQ
              Matches when the masked argument value is equal to the masked
              datum value, example:

              SCMP_CMP( arg , SCMP_CMP_MASKED_EQ , mask , datum )
```

This suggests that the mask is the first value, not the second.

**Comment by pcmoore** [comment]
Thanks @debfx.  We're busy at the moment trying to get libseccomp v2.5.0 released but the gloang bindings are going to be an area of focus after that release.

**Comment by kolyshkin** [comment]
@pcmoore @drakenclimber I believe this one is being fixed by #81.

**Comment by pcmoore** [comment]
Fixed in #81
